src/EncompassRest/Loans/Enums/CondoClass.cs

```diff
@@ -0,0 +1,16 @@
+using System.Runtime.Serialization;
+
+namespace EncompassRest.Loans.Enums
+{
+    public enum CondoClass
+    {
+        [EnumMember(Value = "1")]
```

Comment:
numeric value to match string value

Reply:
In case you didn't see I've done as you suggested.